oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -453,6 +469,13 @@ public Builder setHttpTransportFactory(HttpTransportFactory transportFactory) {
 
     public Builder setScopes(Collection<String> scopes) {
       this.scopes = scopes;
+      this.defaultScopes = null;
+      return this;
+    }
+
+    public Builder setScopes(Collection<String> scopes, Collection<String> defaultScopes) {
```

Comment:
This method is not needed, unless it's required for an interface. Similar to with AppEngineCredentials, the `defaultScopes` should only be seen as an argument to `createScoped`

Reply:
done. This only applies to `ComputeEngineCredentials`. `AppEngineCredentials` doesn't have a Builder class, so it only uses `defaultScopes` in `createScoped`.